sdk/common/shared/src/test/scala/org/typelevel/otel4s/sdk/scalacheck/Gens.scala

```diff
@@ -41,6 +43,17 @@ trait Gens extends org.typelevel.otel4s.scalacheck.Gens {
       attributes <- Gens.attributes
     } yield InstrumentationScope(name, version, schemaUrl, attributes)
 
+  val traceContext: Gen[TraceContext] =
+    for {
+      traceId <- Gen.zip(Gen.long, nonZeroLong)
+      spanId <- nonZeroLong
+      sampled <- Gen.oneOf(true, false)
```

Comment:
is there no `Gen.boolean` already?

Reply:
I've replaced `Gen.oneOf(true, false)` with `Arbitrary.arbitrary[Boolean]`.